README.md

```diff
@@ -30,6 +30,7 @@ on:
 
 jobs:
   main:
+    name: check_pr_title
```

Comment:
What do you think about using something human readable like it is suggested in the [GH actions docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-5)?

```suggestion
    name: Validate PR title
```

Reply:
Sure - I rebased off of current trunk, and updated the name to match your suggestion.